src/rendering/src/services/SendService.tsx

```diff
@@ -0,0 +1,75 @@
+import Script from 'next/script';
+
+const SEND_WEBSITE_ID = process.env.NEXT_PUBLIC_SEND_WEBSITE_ID || '';
+export const isSendConfigured = !!SEND_WEBSITE_ID;
+let isSendInitialized = false;
+
+declare global {
+  interface Window {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    mootrack: any;
+  }
+}
+
+export const SendScripts: JSX.Element | undefined = isSendConfigured ? (
+  <>
+    <Script id="sendTracker">{`
+      //load TrackerJS
+      !function(t,n,e,o,a){function d(t){var n=~~(Date.now()/3e5),o=document.createElement(e);o.async=!0,o.src=t+"?ts="+n;var a=document.getElementsByTagName(e)[0];a.parentNode.insertBefore(o,a)}t.MooTrackerObject=a,t[a]=t[a]||function(){return t[a].q?void t[a].q.push(arguments):void(t[a].q=[arguments])},window.attachEvent?window.attachEvent("onload",d.bind(this,o)):window.addEventListener("load",d.bind(this,o),!1)}(window,document,"script","//cdn.stat-track.com/statics/moosend-tracking.min.js","mootrack");
+      `}</Script>
+  </>
+) : undefined;
+
+export function initialize(): void {
```

Comment:
It does not look like user story 8500 was implemented in the Edge repo before the code was ported to XM Cloud as the 8472 user story description mentions. I will mark this PR with "changes requested".

Reply:
Updated the code to match Edge (do not track in Edit/Preview modes)